web/modules/custom/bnf/tests/src/Kernel/BnfMapperManagerTest.php

```diff
@@ -68,10 +75,6 @@ public function testBnfArticleMapper(): void {
     $this->assertSame($node, $nodeProphecy->reveal());
     $nodeProphecy->set('title', 'Bibliotekarerne anbefaler læsning til den mørke tid')->shouldHaveBeenCalled();
     $nodeProphecy->set('field_paragraphs', [$paragraphProphecy->reveal()])->shouldHaveBeenCalled();
-    $paragraphProphecy->set('field_body', [
```

Comment:
This assertion has been removed, as I place the code directly in the  `create` method, rather than dedicated `set`'s in the mapper.